push_to_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# The image build step occurs only when a build is triggered by a commit being merged into the "master" branch.
+# Given that the "master" branch is protected, the only way to trigger a build from the "master" branch is when a PR
+# is merged into it. This way, we avoid storing images from non-stable branches.
+VERSION=$(cat ./version/version.go | grep "var VERSION" | awk ' { print $4 } ' | sed s/\"//g)
+VERSION_REGEX="^$VERSION$"
+TAG=$(curl https://registry.hub.docker.com/v1/repositories/tfgco/khan/tags | \
+    sed -e 's/[][]//g' -e 's/"//g' -e 's/ //g' | \
+    tr '}' '\n'  | \
+    awk -F: '{print $3}' | \
+    grep "$VERSION_REGEX"
+)
+
+if [ "$TAG" = "$VERSION" ]; then
+  echo "An image with this version already exists in Docker Hub. Please update your version.go file and try again."
+fi
+
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+
+docker build -t will-iam .
+docker tag will-iam:latest tfgco/will-iam:$VERSION
```

Comment:
This is very likely overwriting `"tfgco/will-iam:$VERSION"`, which I don't think we want.

Reply:
Looking at the [documentation](https://docs.docker.com/engine/reference/commandline/tag/) this should create a new tag `tfgco/will-iam:$VERSION` which references `will-iam:latest`(local version). So no overwrite should happen here 🙏